certification/internal/shell/less_than_max_layers.go

```diff
@@ -1,16 +1,71 @@
 package shell
 
 import (
+	"encoding/json"
+	"os/exec"
+
+	"github.com/itchyny/gojq"
 	"github.com/komish/preflight/certification"
-	"github.com/komish/preflight/certification/errors"
 	"github.com/sirupsen/logrus"
 )
 
-type UnderLayerMaxCheck struct {
-}
+var acceptableLayerMax = 40
```

Comment:
Any reason why we don't put this inside the UnderLayerMaxCheck struct ?

Reply:
It is possible technically, however this is not the property of this struct. It is a constant number which is set in advance (not by caller); similar to a constant static field in Java :)
Do you see any benefit in adding this as the type?

Reply:
Mainly to make it programmatically configurable but I guess we don't want this for our use case. Make sense to leave it as a constant. 👍

Reply:
Thanks Sid for taking the time to review this! Very good point!